scipy/stats/tests/test_stats.py

```diff
@@ -3001,6 +3001,27 @@ def _assert_equal(self, actual, expect, *, shape=None, dtype=None):
             dtype = expect.dtype
         assert actual.dtype == dtype
 
+    def test_moment_center(self):
+        # mean((testcase-mean(testcase))**power,axis=0),axis=0))**power))
+        y = stats.moment(self.testcase, 1, center=0)
+        assert_approx_equal(y, 2.5)
+        y = stats.moment(self.testcase, 1, center=2)
```

Comment:
To avoid repetition, please use [`pytest.mark.parametrize`](https://docs.pytest.org/en/6.2.x/parametrize.html).